### DISCUSSION TITLE
Backup iOS sidecars from app

### DISCUSSION THREAD

**Discussion by ericswpark** [immich-app/immich#15925] [Feature Request]
### The bug

Modifications to photos with sidecar files (`.xmp`) are apparently not getting backed up by the iOS app on Immich. This includes data like photo modification (crop, color correction, etc.) and other edited metadata (like manually-tagged GPS coordinates, etc.)

### The OS that Immich Server is running on

unRAID 6.11.5

### Version of Immich Server

1.50.1

### Version of Immich Mobile App

1.50.0 build.88

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
N/A, using monolithic image from https://github.com/imagegenius/docker-immich
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
1. As a simple repro, take any photo without GPS data on iOS
2. Manually add GPS data to it with the Photos app
3. Back up the photo with Immich
4. The photo shows up without any GPS metadata
```

### Additional information

I'm not sure whether iOS actually allows third-party apps to view sidecar files. In that case, the documentation should probably note this caveat so users don't end up losing their manually-edited metadata.

Possibly related to issue https://github.com/immich-app/immich/issues/552#issuecomment-1338334083

**Comment by alextran1502** [comment]
I don't believe this supported yet

**Comment by ericswpark** [comment]
@alextran1502 what would be needed to implement this on Immich? This is preventing me from trying out Immich, as I need my photos in original quality with all edits made to it preserved using a sidecar file.

Can Immich accept additional files as "attachments" to the original file, so that when an image is viewed, Immich checks if the image has a sidecar "attachment" and transparently apply that on top before serving the photo to the user?

**Comment by stevenwdhu** [comment]
I think it's important to read sidecar metadata too. I'm trying to back up my photos from iCloud to Immich, but there are thousands of pictures without metadata(I saved them on my iPhone) showing up as created today in Immich. Actually, they show the correct date they were saved in iCloud Photos, which is what I want. 

It's super annoying because manually editing the metadata for each photo is just impossible. So if Immich could read the metadata from the Photo Library of iCloud Photos, it would help a lot for Apple users.

**Comment by jrasm91** [comment]
Sidecar support for immich has been added to the server and the CLI. It seems reasonable that the mobile app could be updated to do the same.

#2466
